src/main/java/core/basesyntax/db/StorageDaoImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.db;
+
+import java.util.Set;
+
+public class StorageDaoImpl implements StorageDao {
+    public StorageDaoImpl() {
+    }
+
+    @Override
+    public void put(String fruit, Integer quantity) {
+        Storage.fruits.put(fruit,quantity);

```

Comment:
There is no space after the comma in the put method parameters. Code style suggests there should be a space after each comma for better code readability.

Reply:
ok, done